packages/custom_lint_visitor/pubspec.yaml

```diff
@@ -7,7 +7,7 @@ environment:
   sdk: ">=3.0.0 <4.0.0"
 
 dependencies:
-  analyzer: 6.11.0
+  analyzer: ^7.0.0
```

Comment:
A tight version is mandatory here.
And you need to re-run the generator

Reply:
I don't see why a Freezed version is necessary for custom_lint

Reply:
Because freezed also depends on anaylzer...